backbone/docker-compose.yml

```diff
@@ -4,7 +4,11 @@ version: "3.6"
 
 services:
   rabbitmq:
-    image: ${RABBITMQ_IMAGE}:${RABBITMQ_VERSION}
+    build:
```

Comment:
Don't use that here, if needed create a repos RabbitMQ on MediaCloudAI community.